packages/plugin-core/src/views/TreeViewConfig.ts

```diff
@@ -0,0 +1,20 @@
+import { TreeViewItemLabelTypeEnum } from "@dendronhq/common-all";
+import { MetadataService } from "@dendronhq/engine-server";
+import { ITreeViewConfig } from "../common/ITreeViewConfig";
+
+/**
+ * Config for Tree View when extension is run locally- this version pull values from
+ * MetadataService
+ */
+export class TreeViewConfig implements ITreeViewConfig {
```

Comment:
Can we move this to `src\views\node\treeview` and then rename the file/class to `MetadataSvcTreeViewConfig`?